### ISSUE TITLE
:next and :previous options to revert_to

### ISSUE THREAD

**Issue by lailsonbm** [laserlemon/vestal_versions#12]
I suggest to support the options :next and :previous on revert_to. It would be very simple to implement and make the code sweeter! =D

**Comment by laserlemon** [comment]
In the 1.0 branch, you have the ability to extend the `has_many :versions` association with a module or block. There you can easily define `next` and `previous` methods which can be used for reversion via a symbol argument.

Additionally, instead of repeating the `:extend => NextAndPreviousMixin` option in every model, you have the option to use a global configuration block in an initializer:

```
VestalVersions.configure do |config|
  config.class_name = "MyOwnVersionModel"
  config.extend = NextAndPreviousMixin
end
```

As these two specific extensions would be very straightforward and useful, I'll consider merging them into 1.0 development. I see you've forked and committed this addition. Could you please be sure that your change will apply cleanly to the "integration" branch? Thank you!

**Comment by lailsonbm** [comment]
Hi laserlemon. Thanks for you attention and quick reply.
In my fork, I just included for the master branch and I also made some basic tests. At this version, it is working well.
Now, I'll try to integrate it at the _integration_ branch. And wow, vestal_versions is pretty advanced there. Congrats for your work!

**Comment by laserlemon** [comment]
I've decided not to include this specific functionality in the (just-released) 1.0 release. However (as mentioned above), better ability to extend the `has_many :versions` association has been added and it should be trivial to add this functionality per model or application.

The main reason for excluding this from the 1.0 release is that initial versions are no longer created when a versioned model is created. This means that reverting to `previous` may produce unexpected results if sitting on version 2 and no version 1 exists.
